src/services/services.ts

```diff
@@ -1967,6 +1967,77 @@ namespace ts {
         }
     }
 
+    function isObjectLiteralElement(node: Node): node is ObjectLiteralElement  {
+        switch (node.kind) {
+            case SyntaxKind.JsxAttribute:
+            case SyntaxKind.JsxSpreadAttribute:
+            case SyntaxKind.PropertyAssignment:
+            case SyntaxKind.ShorthandPropertyAssignment:
+            case SyntaxKind.MethodDeclaration:
+            case SyntaxKind.GetAccessor:
+            case SyntaxKind.SetAccessor:
+                return true;
+        }
+        return false;
+    }
+
+    function getNameFromObjectLiteralElement(node: ObjectLiteralElement) {

```

Comment:
>getNameFromObjectLiteralElement [](start = 13, length = 31)

I think getTextOfPropertyName from compiler's utilities is exactly the same as this code.